source/dem/particle_wall_dmt_force.cc

```diff
@@ -9,87 +9,9 @@ template <int dim>
 ParticleWallDMTForce<dim>::ParticleWallDMTForce(
   const DEMSolverParameters<dim>       &dem_parameters,
   const std::vector<types::boundary_id> boundary_index)
-  : ParticleWallContactForce<dim>(dem_parameters)
+  : ParticleWallNonLinearForce<dim>(dem_parameters)
+  , dmt_cut_off_distance(set_dmt_cut_off_distance(dem_parameters))
```

Comment:
does this need to be a member variable? I would move it closer to the calculation loop. Avoid making member variable when possible IMO

Reply:
I think the same solution can be apply to particle-wall contacts.